content/english/anti-vegan/trapped-on-an-island-with-a-pig.md

```diff
@@ -16,12 +16,12 @@ contributors:
         name: deb-hardwicke
         role: proofreader
 draft: false
-description: "If it's just you and a pig on a desert island surely you would eat the pig to survive?"
+description: "How to respond to the argumnent: If it's just you and a pig on a desert island surely you would eat the pig to survive?"
```

Comment:
Typo: `argumnent`

Reply:
Yeah noticed that 15 seconds after hitting commit. DoH!